graphql/util.go

```diff
@@ -55,3 +57,20 @@ func SetScheduled(ctx context.Context, sc data.Connector, taskID string, isActiv
 	}
 	return &apiTask, nil
 }
+
+// GetFormattedDate returns a time.Time type in the format "Dec 13, 2020, 11:58:04 pm"
+func GetFormattedDate(t *time.Time, timezone string) (*string, error) {
+	if t == nil {
+		return nil, nil
+	}
+
+	loc, err := time.LoadLocation(timezone)
+	if err != nil {
+		return nil, err
+	}
+
+	timeInUserTimezone := t.In(loc)
+	newTime := fmt.Sprintf("%s %d, %d, %s", timeInUserTimezone.Month(), timeInUserTimezone.Day(), timeInUserTimezone.Year(), timeInUserTimezone.Format(time.Kitchen))
```

Comment:
Use the time.Format function and pass in the right format string, instead of creating your own format. This format is also not locale-friendly for any locale outside en-US

Reply:
i tried using time.Format but a go format string for this format does not exist